src/utils/dbUtils.ts

```diff
@@ -60,22 +60,25 @@ export const saveToDb = (
 }
 
 export const getFromDb = async (
-  localDb: any,
+  localDb: nano.DocumentScope<DbDoc>,
   dates: string[]
 ): Promise<DbDoc[]> => {
   // Grab existing db data for requested dates
-  const documents = await Promise.all(
-    dates.map(date => {
-      return localDb.get(date).catch(e => {
-        if (e.error !== 'not_found') {
-          slackPoster(config.slackWebhookUrl, e).catch(e)
-        } else {
-          return {
-            _id: date
-          }
-        }
-      })
-    })
-  )
-  return documents
+  const response = await localDb.fetch({ keys: dates }).catch(e => {
+    if (e.error !== 'not_found') {
+      // slackPoster(config.slackWebhookUrl, e).catch(e)
```

Comment:
why is this commented out?

Reply:
Commented out so my dumb mistakes while testing wouldn't spam slack. Fixed.